.gitignore

```diff
@@ -7,6 +7,10 @@
 /cmake
```

Comment:
Also suggest /cmake* as many times there are variations of the cmake folder being used.